### PR TITLE
docs(aio): update sidenav layout, remove corresponding files

### PR DESCRIPTION
**Pull request by kapunahelewong**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe: update sidenav layout/order
```

**What is the current behavior?** (You can also link to an open issue here)
Current order of nav items.

**What is the new behavior?**
New organization of nav items. Remove changelog, quickstart, webpack, update sidenav, upd…  …
…ate tutorial intro, rename/move appmodule

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
There is a placeholder nav item for Common Templating Patterns, Expressions and Statements, and Forms Overview.

### CODE REVIEW

File: aio/content/guide/bootstrapping.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 @title
-AppModule: the root module
+Bootstrapping
 
 @intro
-Tell Angular how to construct and bootstrap the app in the root "AppModule".
+Tell Angular how to construct and bootstrap the app in the root `AppModule`.
```
Comment (petebacondarwin) [inline aio/content/guide/bootstrapping.md]: You should remove all `@intro` blocks now, which then leads to the fact that you can also move the `@title` block to an h1 tag and then drop `@title` and `@description` too.

Comment (kapunahelewong) [inline aio/content/guide/bootstrapping.md]: Thanks, @petebacondarwin! So they will change from:

`@title
Bootstrapping

@intro
Tell Angular how to...

@description
`

to just: 

`# Bootstrapping`

?

Comment (petebacondarwin) [inline aio/content/guide/bootstrapping.md]: Right! :-)

File: aio/content/navigation.json [hunk 2]
```diff
@@ -350,49 +310,28 @@
...
           "url": "guide/glossary",
           "title": "Glossary",
           "tooltip": "Brief definitions of the most important words in the Angular vocabulary."
         },
-
         {
-          "url": "guide/styleguide",
-          "title": "Style Guide",
-          "tooltip": "Write Angular with style."
+          "url": "guide/cheatsheet",
```
Comment (IgorMinar) [inline aio/content/navigation.json]: can we move fundamentals to the foundations? maybe as the last item there?

File: aio/content/navigation.json [hunk 3]
```diff
@@ -27,23 +27,13 @@
     },
 
     {
+      "url": "guide/cli-quickstart",
```
Comment (IgorMinar) [inline aio/content/navigation.json]: can we rename the doc to be called just "quickstart" ?

File: aio/content/navigation.json [hunk 4]
```diff
@@ -148,24 +105,88 @@
...
+              "title": "User Input",
+              "tooltip": "User input triggers DOM events. We listen to those events with event bindings that funnel updated values back into our components and models."
+            },
+            {
+              "url": "guide/forms",
+              "title": "Template-driven Forms",
+              "tooltip": "A form creates a cohesive, effective, and compelling data entry experience. An Angular form coordinates a set of data-bound user controls, tracks changes, validates input, and presents errors."
+            },
+            {
+              "url": "guide/dynamic-form",
```
Comment (IgorMinar) [inline aio/content/navigation.json]: this is one depends on reactive forms, can we move it just after reactive forms?

File: aio/content/navigation.json [hunk 5]
```diff
@@ -148,24 +105,88 @@
...
-              "title": "Component Styles",
-              "tooltip": "Learn how to apply CSS styles to components."
-            },
-            {
               "url": "guide/dynamic-component-loader",
               "title": "Dynamic Components",
               "tooltip": "Load components dynamically."
             },
             {
+              "url": "guide/template-syntax",
```
Comment (IgorMinar) [inline aio/content/navigation.json]: can we move this just after displaying-data?

File: aio/content/navigation.json (hunk 3 above)
Comment (kapunahelewong) [inline aio/content/navigation.json]: Done.

File: aio/content/navigation.json (hunk 5 above)
Comment (kapunahelewong) [inline aio/content/navigation.json]: Done.

File: aio/content/navigation.json (hunk 4 above)
Comment (kapunahelewong) [inline aio/content/navigation.json]: Done.

File: aio/content/navigation.json (hunk 2 above)
Comment (kapunahelewong) [inline aio/content/navigation.json]: @IgorMinar I'm not sure where Foundations is. Could you clarify? Thanks!

Comment (IgorMinar) [inline aio/content/navigation.json]: sorry!!! I meant, can we move the cheatsheet into the fundamentals section as the last item there.

Comment (kapunahelewong) [inline aio/content/navigation.json]: @IgorMinar Aha! Sure, thanks! :)

**Comment by IgorMinar** [comment]
Shouldn't "Getting Started" be in all caps to be consistent with the rest of the top level menu items?
screen shot 2017-05-24 at 12 31 26 pm

**Comment by petebacondarwin** [comment]
That is a CSS issue - currently only "container" nodes are given upper case style.

**Comment by IgorMinar** [comment]
@petebacondarwin "API" is not a container element. Is it just that it happens to be all caps in the source form?

**Comment by petebacondarwin** [comment]
Precisely

File: packages/examples/platform-browser/browser/title.ts~HEAD [hunk 6]
```diff
@@ -0,0 +1,26 @@
+// #docplaster
```
Comment (petebacondarwin) [inline packages/examples/platform-browser/browser/title.ts~HEAD]: Is the filename correct here?

Comment (kapunahelewong) [inline packages/examples/platform-browser/browser/title.ts~HEAD]: @petebacondarwin Oh my goodness, thank you for spotting that.

**Comment by IgorMinar** [comment]
I rebased this PR and force-pushed into this branch

**Comment by petebacondarwin** [comment]
I have assumed that the `title` example was added by mistake but then instead of being deleted the actual `Title` service source file was deleted. I am reverting both of those from this PR, since I believe they are not relevant and we can easily reproduce later.

Also I have removed all links from sidenav container nodes, since the UI is no longer designed to support that; and the urls to which they linked are the first items of their children anyway.

Updated and force pushed

**Review by petebacondarwin — APPROVED** [review]

**Pull request merged**